scripts/docker_build/sof_builder/Dockerfile

```diff
@@ -64,13 +64,15 @@ RUN useradd --create-home -d /home/sof -u $UID -G sudo sof && \
 echo "sof:test0000" | chpasswd && adduser sof sudo
 ENV HOME /home/sof
 
+ARG XT_OVERLAY_REPO=https://github.com/thesofproject/xtensa-overlay.git
+ARG CT_NG_REPO=https://github.com/thesofproject/crosstool-ng.git
```

Comment:
To ease local mirroring, is it possible to have something like:
```
CT_NG_REPO=$GITHUB_SOF/crosstool-ng.git
```

Reply:
What do you mean by local mirroring?

Reply:
Exactly as it says. Use a single variable for GITHUB_SOF path, so that for example you want to change to use a local mirror you only need to change GITHUB_SOF variable instead of going places and changing all over the script file.

Reply:
Sorry for the delay, I've amended to reflect these changes